clarity/src/vm/tests/simple_apply_eval.rs

```diff
@@ -1424,3 +1426,46 @@ fn test_is_mainnet() {
             )
         });
 }
+
+#[test]
+fn test_chain_id() {
+    let tests = ["chain-id"];
+
+    let mainnet_expectations = [Value::UInt(CHAIN_ID_MAINNET.into())];
+
+    tests
+        .iter()
+        .zip(mainnet_expectations.iter())
+        .for_each(|(program, expectation)| {
+            assert_eq!(
+                expectation.clone(),
+                execute_with_parameters(
+                    program,
+                    ClarityVersion::Clarity2,
+                    StacksEpochId::Epoch20,
```

Comment:
why not Epoch21? same as below.

Reply:
Ah, that's an oversight on my end.  Will fix.